rest/src/test/java/com/carhouse/rest/controllerIT/CarControllerTestIT.java

```diff
@@ -0,0 +1,125 @@
+package com.carhouse.rest.controllerIT;
+
+import com.carhouse.model.*;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.RestTemplate;
+
+import java.sql.Date;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CarControllerTestIT {
+
+    private static final String HOST = "http://localhost:8080";
```

Comment:
bad idea to use port `8080` for tests. maybe it already in use by someone on PC.

Reply:
I've changed port